api/package.json

```diff
@@ -29,6 +29,7 @@
     "redis": "^4.6.14",
     "tsx": "^4.12.0",
     "winston": "^3.13.0",
+    "xmlbuilder2": "^3.1.1",
```

Comment:
I strongly recommend using the `sitemap` package instead of hand-writing your own XML structure